examples/mpu6050/main.go

```diff
@@ -11,12 +11,27 @@ import (
 func main() {
 	machine.I2C0.Configure(machine.I2CConfig{})
 
-	accel := mpu6050.New(machine.I2C0)
-	accel.Configure()
+	mpuDevice := mpu6050.New(machine.I2C0, mpu6050.DefaultAddress)
```

Comment:
In other drivers, this has been solved by exporting `Address` and then just setting it before calling `Configure`. That way we do not have to break the existing API and all of the many projects that use this driver. I have several myself.

```go
mpuDevice := mpu6050.New(machine.I2C0)
mpuDevice.Address = 0x66 // overrides default

// now do everything else
```